docs/generated/manifests/nx.json

```diff
@@ -6228,14 +6238,24 @@
   },
   "/deprecated/workspace-layout": {
     "id": "workspace-layout",
-    "name": "workspace-layout",
+    "name": "workspaceLayout",
     "description": "",
     "file": "shared/deprecated/workspace-layout",
     "itemList": [],
     "isExternal": false,
     "path": "/deprecated/workspace-layout",
     "tags": []
   },
+  "/deprecated/name-and-directory-format": {
+    "id": "name-and-directory-format",
+    "name": "nameAndDirectoryFormat",
```

Comment:
I would change this name to something more like...

"As Provided vs Derived"

Reply:
There is already the [workspaceLayout](https://nx.dev/deprecated/workspace-layout) deprecation guide.  I can combine this guide with that one.